crates/rattler_pypi_interop/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#![deny(missing_docs, dead_code)]
+
+//! This crate provides a set of functions and data types for working with Python packages installed via
+//! `PyPI` compatible installers (e.g. `pip`, `uv`, `poetry`, etc.).
+//!
+//! Some of the things you can do with this crate include:
+//!
+//! - Querying a `PyPI` repository for package metadata.
+//! - Reading Python package metadata from locally installed packages.
+//!
```

Comment:
I think it would be especially good to amend this with what the library wont do!